src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+			</li>
+			<li>
+				<p>Controller - Added short integer “CodeId” property, which will replace the “Code” property. New “AnyController” for J1708 engine diagnostics to allow replacing those engine diagnostics identical except for the controller with one diagnostic. J1708 engine diagnostics for 58 separate SIDs were updated.</p>
+			</li>
+			<li>
+				<p>CustomDevice - Added “VehicleIdentificationNumber” property.</p>
+			</li>
+			<li>
+				<p>FuelTaxDetail - Added “Driver” property.</p>
+			</li>
+			<li>
+				<p>DriverChangeSearch - Added “IncludeOverlappedChanges” property - A value indicating whether to include the last driver change before the from date or the most recent driver change (if the from date is not provided).</p>
+			</li>
+			<li>
+				<p>InvalidUserException - Message changed from to “Incorrect MyGeotab login credentials…” to “Incorrect login credentials…”.</p>
+			</li>
+		</ul>
+		<h2>New Objects</h2>
+		<ul>
+			<li>
+				<p>DutyStatusMalfunctionType - Added - Malfunction or Diagnostic type of the DutyStatusLog.</p>
+			</li>
+			<li>
+				<p>DutyStatusState - Added - The record status of DutyStatusLog.</p>
+			</li>
+		</ul>
+	</div>
+);
+
+const update1707: ReactNode = (
+	<div className="paragraph">
+		<ul>
+			<li>
+				<p>Web Request Notifications: fix {'{'}zoneId{'}'} and {'{'}zoneComment{'}'} tokens would not get populated unless {'{'}zone{'}'} or {'{'}address{'}'} were also included.</p>
+			</li>
+			<li>
+				<p>Documentation: API Reference updated to include default value and max length of object properties in their descriptions. Default values are automatically used when adding an entity and those properties have no value assigned (are null). For example, adding a Group with color = null, will add group with default color “Blue”. If a property has no default value, it is required when calling “Add”.</p>
+			</li>
+			<li>
+				<p>Set operations now retain the value of missing (null) properties. A positive effect of this change is that is remedies a long existing issue that could occur when a server is a newer version (ex 5.7.1704) than client nuget package (ex 5.7.1701) which is making requests to it. The issue could arise when a new Enum value was added and exists only in the server’s code base, not client client nuget package. When the unknown Enum was received by the client, it could not be deserialiezed into an Enum value and would throw an exception. Starting in nuget package version 5.7.1707.x, the unknown Enum value will be deserialiezed to null. This means the object can round trip” on “Set” because the server will now (starting at version 5.7.1707) fill in the null value with the existing saved value of the property.</p>
```

Comment:
The character U+2019 "’" could be confused with the ASCII character U+0060 "`", which is more common in source code.

Also typo "deserialiezed"